infra/modules/database/role_manager/role_manager.py

```diff
@@ -0,0 +1,134 @@
+import itertools
+from operator import itemgetter
+import os
+import logging
+from pg8000.native import Connection, identifier
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+def lambda_handler(event, context):
+    conn = connect()
+
+    logger.info("Current database configuration")
+
+    prev_roles = get_roles(conn)
+    print_roles(prev_roles)
+
+    prev_schema_privileges = get_schema_privileges(conn)
+    print_schema_privileges(prev_schema_privileges)
+
+    logger.info("Configuring database")
+    configure_database(conn)
+
+    logger.info("New database configuration")
+
+    new_roles = get_roles(conn)
+    print_roles(new_roles)
+
+    new_schema_privileges = get_schema_privileges(conn)
+    print_schema_privileges(new_schema_privileges)
+
+    return {
+        "roles": new_roles,
+        "roles_with_groups": get_roles_with_groups(conn),
+        "schema_privileges": {
+            schema_name: schema_acl
+            for schema_name, schema_acl
+            in new_schema_privileges
+        },
+    }
+
+def connect() -> Connection:
+    user = os.environ.get("DB_USER")
+    host = os.environ.get("DB_HOST")
+    port = os.environ.get("DB_PORT")
+    password = os.environ.get("DB_PASSWORD")
+
+    logger.info("Connecting to database: user=%s host=%s port=%s", user, host, port)
+    return Connection(user=user, host=host, port=port, password=password)
+
+
+def get_roles(conn: Connection) -> list[str]:
+    return [row[0] for row in conn.run("SELECT rolname \
+                                       FROM pg_roles \
+                                       WHERE rolname NOT LIKE 'pg_%'\
+                                       AND rolname NOT LIKE 'rds%'")]
+
+
+def get_roles_with_groups(conn: Connection) -> list[tuple[str, str]]:
```

Comment:
Return type doesn't match the code:

```suggestion
def get_roles_with_groups(conn: Connection) -> dict[str, str]:
```

Reply:
thanks!